napari/_qt/widgets/qt_large_int_spinbox.py

```diff
@@ -7,7 +7,20 @@
 logger = logging.getLogger(__name__)
 
 
-class ArbitraryIntDoubleSpinBox(QDoubleSpinBox):
+class QtLargeIntSpinBox(QDoubleSpinBox):
+    """
+    A class for integer spinboxes backed by a double (i.e. float64).
+
+    Qt's built-in ``QSpinBox`` is backed by a signed 32-bit integer.
+    This could become limiting, particularly in large dense segmentations.
+    This class behaves like a ``QSpinBox`` backed by a signed 54-bit integer.
```

Comment:
```suggestion
    This class behaves like a ``QSpinBox`` backed by a signed 64-bit integer.
```

Typo? Should this be `64-bit`?

Reply:
54-bit is (very nearly) correct as it's using a 64-bit float, which can encode all integers in the range `[-2**53, 2**53]` - it can of course encode larger numbers and fractions but not all of the integers outside that range. A signed 54-bit int would be able to encode `(-2**53, 2**53)`.

Reply:
Oh, ok great! Thanks for the extra explantation!!